pkg/crd/spec.go

```diff
@@ -151,6 +151,19 @@ func (p *Parser) NeedCRDFor(groupKind schema.GroupKind, maxDescLen *int) {
 		packages[0].AddError(fmt.Errorf("CRD for %s has no storage version", groupKind))
 	}
 
+	served := false
+	for _, ver := range crd.Spec.Versions {
+		if ver.Served {
+			served = true
+			break
+		}
+	}
+	if !served {
+		// just add the error to the first relevant package for this CRD,
+		// since there's no specific error location
+		packages[0].AddError(fmt.Errorf("CRD for %s does not serve any version", groupKind))
```

Comment:
WDYT about something as:

```suggestion
		packages[0].AddError(fmt.Errorf("CRD for %s has the vesion %s which is no longer supported", groupKind, crdVersion ))
```